### ISSUE TITLE
Heavy impact on bundle size

### ISSUE THREAD

**Issue by framefactory** [koj-co/types#87]
When importing certain types from this package, my bundle size jumps from 167k to 402k. Specifically, adding the following import causes the bundle size to increase (it also happens with other imports):
```
import { Apartment } from "@koj/types";
```
Is this intended behavior?
I'd expect to only import type information, which should not affect bundle size.

**Comment by framefactory** [comment]
Workaround: above size impact can be avoided by explicitly importing type information only, like so:
```
import type { Apartment } from "@koj/types"
```

**Comment by AnandChowdhary** [comment]
I agree, in general, one should use `import type` when importing only type data, because otherwise TypeScript compiles these as values (e.g., enums become objects) which is likely what is adding these extra bytes for you. Let me see if I can export them in a way that does this by default.

**Comment by AnandChowdhary** [comment]
The 235 KB is still super high though, most likely due to Class Validator. I'll check it out!
